fe1-web/__tests__/model/network/FromJsonData.test.ts

```diff
@@ -456,7 +533,7 @@ describe('=== fromJsonData checks ===', () => {
 
     it('should fail when omitting a mandatory parameter', () => {
```

Comment:
it seems that this section, described as `should fail (throw) during object creation` , really only tests `CreateLao`.
I'd suggest renaming it to `should fail (throw) during CreateLao creation`
And split your tests apart in another describe, dealing only with the `SetupElection` object.

Reply:
It also tests `stateLao`, `createMeeting` and `WitnessMessage` in the same describe. Should I put all of them in their own describe?